client/Bootstrap/index.ts

```diff
@@ -1,3 +1,7 @@
+/*
```

Comment:
odd that this was appended again. Will circle back around, but was the licence tool updated to expect .ts files?

Reply:
I think I hit this in my test pr too. Should be fixed now